app/services/stockit/orders_package_sync.rb

```diff
@@ -0,0 +1,105 @@
+require 'stockit/base'
+
+module Stockit
+  class OrdersPackageSync
+
+    include Stockit::Base
+
+    attr_accessor :package, :errors, :offset, :per_page, :orders_package
+
+    def initialize(package, orders_package, offset = nil, per_page = nil)
+      @package = package
+      @orders_package = orders_package
+      @errors = {}
+      @offset = offset
+      @per_page = per_page
+    end
+
+    class << self
+      def create(package, orders_package)
+        new(package, orders_package).create
+      end
+
+      def update(package, orders_package)
+        new(package, orders_package).update
+      end
+
+      def delete(package, orders_package)
+        new(package, orders_package).delete
+      end
+    end
+
+    def create
+      if package.inventory_number.present?
+        url = url_for("/api/v1/items")
+        post(url, stockit_params.merge!({generate_q_inventory_number: true}))
+      end
+    end
+
+    def update
+      if package.inventory_number.present?
+        url = url_for("/api/v1/items/update")
+        put(url, stockit_params.merge!({update_gc_orders_package: true, orders_package_id: @orders_package.id, orders_package_state: @orders_package.state}))
+      end
+    end
+
+    def delete
+      url = url_for("/api/v1/items/destroy")
+      put(url, { gc_orders_package_id: @orders_package.id })
+    end
+
+    private
+
+    def add_stockit_prefix(inventory_number)
+      return "X#{inventory_number}" if !!((inventory_number || "")[0..0] =~ /[0-9]/)
```

Comment:
Could we define a global constant for stockit prefix "X" so we can refer to it elsewhere too?